airflow/providers/presto/transfers/presto_to_slack.py

```diff
@@ -66,14 +63,13 @@ def __init__(
         sql: str,
         slack_message: str,
         presto_conn_id: str = 'presto_default',
-        slack_conn_id: str = 'slack_default',
+        slack_conn_id: Optional[str] = None,
```

Comment:
this is a breaking change. Can we avoid it?
I prefer to leave the defaults of the old operators as is. In the new operator we can set new defaults as we like

Reply:
Umm, we can, but I'm not sure how should I tackle this..
The thing is, that using both `slack_conn_id` and `slack_webhook_token`, although supported by the `SlackWbhookHook`, can lead to misunderstandings.
I added a check for this in `SqlToSlackOperator`, that if both are provided, we go out with an AirflowException.
If I leave this as is, with the default initialization of the `slack_conn_id`, if a user provides me a `slack_webhook_token`, it will fail (because both are provided - once by the user, and once by the initialization).

To solve this, I can add a check in `SqlToSlackOperator` that checks if `slack_conn_id` is provided, and the connection actually exists, but again it may lead to a confusing flow, as if `slack_conn_id` is provided and exists, it will be the one used by the `SlackWebhookHook`..

I found it to be easier to provide either one, but idk.. WDYT? Should I add a check whether the connection actually exists? or do you might have some other creative suggestion? :)